pkg/handlers/v1/insert_account_owner_test.go

```diff
@@ -0,0 +1,50 @@
+package v1
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/asecurityteam/asset-inventory-api/pkg/domain"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+func newInsertAccountOwnerHandler(storer domain.AccountOwnerStorer) *AccountOwnerInsertHandler {
+	return &AccountOwnerInsertHandler{
+		LogFn:              testLogFn,
+		StatFn:             testStatFn,
+		AccountOwnerStorer: storer,
+	}
+}
+
+func validInput() AccountOwner {
```

Comment:
info: `validInput()` is slightly confusing as name, it made me think of input validation.

Could we name this something slightly more descriptive to testing? `testAccountOwner`, `testInput` or `testData`?

Reply:
Yes, sure.